connectors/gitlab-pr/src/test/java/com/vmware/connectors/gitlab/pr/GitlabPrControllerTest.java

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright © 2018 VMware, Inc. All Rights Reserved.
```

Comment:
Did you accidentally delete this file? This is showing as a new file.

Reply:
Ya, something went wrong with this file. I had to copy again from master.